core/src/main/resources/error/error-classes.json

```diff
@@ -738,12 +744,24 @@
     ],
     "sqlState" : "42K05"
   },
+  "INVALID_EXTRACT_BASE_FIELD_TYPE" : {
+    "message" : [
+      "Can't extract a value from <base>. Need a complex type [struct, array, map] but got <other>."
```

Comment:
Please, upper case the complex types to be consistent w/ other places, and w/ the function `toSQLType()`.

Reply:
Sounds good. Done.